models/articleModel.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+
+const articleSchema = new mongoose.Schema({
+  title: {
+    type: 'String',
+    required: [true, 'title is required'],
+    unique: [true, 'isnt unique'],
+  },
+  body: {
+    type: String,
+    required: [true, 'article body is required'],
+  },
+  rating: {
+    type: Number,
+    max: 5,
+    min: 0,
+  },
+  author: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'doctorModel',
```

Comment:
Have you tested this! 😅
Remember, we have changed the name of the model to just Doctor

Reply:
done